Jason/Filter_by_Type.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+Python Fundamentals Assignment
+
+Autho: Jason Lee
+
+"""
+from types import *
+
+def filter_by_type(item):
```

Comment:
It's a little overkill to use a type check inside of isinstance. You should be able to get the same functionality by using one or the other.